models/notification.go

```diff
@@ -433,7 +433,13 @@ func (n *Notification) loadComment(e db.Engine) (err error) {
 	if n.Comment == nil && n.CommentID != 0 {
 		n.Comment, err = getCommentByID(e, n.CommentID)
 		if err != nil {
-			return fmt.Errorf("GetCommentByID [%d] for issue ID [%d]: %v", n.CommentID, n.IssueID, err)
+			if IsErrCommentNotExist(err) {
```

Comment:
Isn't it already an `ErrCommentNotExist` in this case? Why construct a new error instead of simply omitting this case then?

Reply:
The old error only include commentID, here we added issueID

Reply:
Adding context is good!